Configuration/Configuration.py

```diff
@@ -75,3 +55,40 @@
 FIRMWARE_LOAD_ADDRESS = 0x40001000      # Firmware will be loaded at this address when using virtual boot
 FIRMWARE_BINARY_FILE_PATH = "CapuaEnvironment/firmware.bin"
 
+REGISTERS = {
+    "A": 0b0000, "B": 0b0001, "C": 0b0010, "D": 0b0011,
+    "E": 0b0100, "F": 0b0101, "G": 0b0110, "S": 0b0111,
+
+    "A2": 0b1000, "B2": 0b1001, "C2": 0b1010, "D2": 0b1011,
+    "E2": 0b1100, "F2": 0b1101, "G2": 0b1110, "S2": 0b1111
+}
+
+INSTRUCTION_LIST = [
+    "ACTI", "ADD", "AND", "CALL", "CMP", "DACTI", "DIV",
+    "HIRET", "INT", "JMP", "JMPR", "MEMR", "MEMW", "MOV",
+    "MUL", "NOP", "NOT", "NOP","OR", "POP", "PUSH",
+    "RET", "SFSTOR", "SIVR", "SHL", "SHR", "SUB", "XOR"
+]
+
+LABEL_INSTRUCTIONS = [
```

Comment:
This is specific to assembler not to VM configuration. I believe the assembler might need an specific "include" on its own. I don't feel this belongs in the main VM configuration file.